.clang-format

```diff
@@ -0,0 +1,168 @@
+---
+Language: Cpp
```

Comment:
Why do we need/want `Language`?

Reply:
This is the default configuration generated using the `clang-format` command. There are [other languages](https://clang.llvm.org/docs/ClangFormat.html
) clang-format formats too btw. I guess it keeps the configuration specific to C++.

Reply:
Removed.